make-get-system-import.js

```diff
@@ -8,6 +8,7 @@ let FILE_USE_DATA = path.join('Views', 'Data.js')
 let FILE_USE_DATA_FORMAT = path.join('Views', 'Data', 'format.js')
 let FILE_USE_DATA_VALIDATE = path.join('Views', 'Data', 'validate.js')
 let FILE_USE_DATA_AGGREGATE = path.join('Views', 'Data', 'aggregate.js')
+let FILE_USE_DATA_COMPARE = path.join('Views', 'Data', 'compare.js')
```

Comment:
I'm not sure if we want to formalise compare. I think we should be able to get away with `data+format` to reduce the value to a boolean, what do you think?

Reply:
makes sense. would you also change the keyword used here to be `format` then instead of `compare`? If the functions are defined in `format.js` maybe better to be `format` I think, otherwise there will be a discrepancy and may lead to confusions

Reply:
Yeah, I wouldn't add compare at all. Eg, if you need a formatter to see if a list has an item, I'd write something like:
```
// format.js
function listWithOneItemToBoolean(value) {
  return list.length === 1
}
```
Is that what you were thinking too?

Reply:
yes, that looks good to me, thanks